databricks_cli/sdk/service.py

```diff
@@ -550,48 +554,50 @@ def export_workspace(self, path, format=None, direct_download=None, headers=None
         if direct_download is not None:
             _data['direct_download'] = direct_download
         return self.client.perform_query('GET', '/workspace/export', data=_data, headers=headers)
-
+    
     def delete(self, path, recursive=None, headers=None):
         _data = {}
         if path is not None:
             _data['path'] = path
         if recursive is not None:
             _data['recursive'] = recursive
         return self.client.perform_query('POST', '/workspace/delete', data=_data, headers=headers)
-
+    
     def get_status(self, path, headers=None):
         _data = {}
         if path is not None:
             _data['path'] = path
         return self.client.perform_query('GET', '/workspace/get-status', data=_data, headers=headers)
-
+     
 
 class SecretService(object):
     def __init__(self, client):
         self.client = client
 
     def create_scope(self, scope, initial_manage_principal=None, scope_backend_type=None,
-                     headers=None):
+                     is_databricks_managed=None, headers=None):
```

Comment:
This shouldn't be here. Maybe you need to merge master before generating the sdk?